docker-compose.yml

```diff
@@ -10,16 +10,16 @@ services:
     environment:
       - VERBOSE=true
       - SYSTEMD_LOGGING=True
-      - matrix_home_server=${MATRIX_HOME_SERVER}
-      - matrix_bot_username=${MATRIX_BOT_USERNAME}
-      - matrix_bot_password=${MATRIX_BOT_PASSWORD}
+      - matrix_home_server=${matrix_home_server}
+      - matrix_bot_username=${matrix_bot_username}
+      - matrix_bot_password=${matrix_bot_password}
       - session_path=/data/session.txt
       - groups_used=["basic", "albert"]
-      - user_allowed_domains=${USER_ALLOWED_DOMAINS}
-      - salt=${SALT}
-      - albert_api_url=${ALBERT_API_URL}
-      - albert_api_token=${ALBERT_API_TOKEN}
-      - albert_api_model_name=${ALBERT_API_MODEL_NAME}
+      - user_allowed_domains=${user_allowed_domains}
+      - salt=${salt}
+      - albert_api_url=${albert_api_url}
+      - albert_api_token=${albert_api_token}
+      - albert_api_model_name=${albert_api_model_name}
     ports:
       - 443:443
       - 80:80
```

Comment:
What do you think of passing all those env variables in MAJ (in .env exemple and docke-compose) to keep that convention ?

Reply:
That'd be way better, but the idea of the PR is to have the same .env file wether you deploy with Docker or out of Docker, given than we use a Pydantic SettingsConfigDict which loads the .env file to map with its keys.
Don't you think it would be good to keep the same env file for Docker and non-Docker runs?

Reply:
Oh good news, just saw that, it's case insensitive: https://docs.pydantic.dev/latest/concepts/pydantic_settings/#case-sensitivity

OK let me change that then

Reply:
Done

Reply:
yes, great !